Dockerfile

```diff
@@ -27,10 +27,11 @@ RUN apt-get update \
      git \
      google-chrome-stable \
      libmagic1 \
-     libpq5 \
+     libpq-dev \
      libxml2 \
      libxmlsec1 \
      libxmlsec1-openssl \
+     make \
```

Comment:
I'm assuming this is now needed because it was included in the docker image base we used previously and is not in the bookworm-slim base?

Reply:
Correct.